change_route/hello.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+)
+
+type Request struct {
+	Name string `json:"name"`
+}
+
+type Response struct {
+	Code      int    `json:"code"`
+	Message   string `json:"message"`
+	Timestamp string `json:"timestamp"`
+}
+
+func HelloHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	var req Request
+	json.NewDecoder(r.Body).Decode(&req)
```

Comment:
scripts directory has build.sh and run.sh in master branch but in this branch it isn't visible, I think it will be visible after merging the pull request.

Reply:
> Please handle the error here.

what kind of error I didn't get your point

Reply:
> scripts directory has build.sh and run.sh in master branch but in this branch it isn't visible, I think it will be visible after merging the pull request.

then align your branch with the latest branch

Reply:
> > Please handle the error here.
> 
> what kind of error I didn't get your point
`json.NewDecoder(r.Body).Decode(&req)` returns error. Please handle it and also check for errors where needed.